demo/b2share_demo/helpers.py

```diff
@@ -291,3 +292,13 @@ def community_id_match(match):
         source
     )
 
+
+def list_db_records():
+    """A generator for all the records in the current instance"""
+    query = RecordMetadata.query.filter(RecordMetadata.json != None)
+    for obj in query.all():
+        record = Record(obj.json, model=obj)
+        if record.get('$schema', "").endswith('draft_json_schema'):
```

Comment:
Why not if record.get("publication_state") is unequal to deposit.PublicationStates.published ?